src/authorization/service/group.service.ts

```diff
@@ -146,6 +164,40 @@ export class GroupService {
     return permissions;
   }
 
+  async updateGroupUsers(id: string, userIds: string[]): Promise<User[]> {
+    await this.getGroupById(id);
+    const usersInRequest = await this.userRepository.findByIds(userIds);
+    const validUsersInRequest: Set<string> = new Set(
```

Comment:
Can we move this validation logic to a private function: `validateUsers(User[])`?

Reply:
usersInRequest and validUsersInRequest are used in the remaining part of code. Should we move it to another function in that case?

Reply:
We will be needing just the valid users in the request right?